[iOS] Undefined symbols for architecture arm64: "_RCTSetLogFunction"

Hello,
I have this linking error:
Undefined symbols for architecture arm64:
  "_RCTSetLogFunction", referenced from:
      -[PropertyFinderTests testRendersWelcomeScreen] in PropertyFinderTests.o

When i try to compile the default project on iOS. (using react-native init)

--- a/local-cli/generator-ios/templates/xcodeproj/project.pbxproj
+++ b/local-cli/generator-ios/templates/xcodeproj/project.pbxproj
@@ -595,6 +595,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				DEAD_CODE_STRIPPING = NO;
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,

